tests/test_dir_layouts.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+from pathlib import Path
+import shutil
+import bids
+from petdeface.petdeface import PetDeface
+from os import cpu_count
+from bids.layout import BIDSLayout
+
+import tempfile
```

Comment:
`pytest` has temp dir fixtures (https://docs.pytest.org/en/6.2.x/tmpdir.html); not sure if that would be more advantageous to use than the `tempfile` based implementation

Reply:
Yeah, pytest has everything, but despite my owning a book on it I'm not wholly familiar. I think we could move to using exclusively pytest, but for the moment it works and I'm able to easily inspect the temp dir if something fails.

Reply:
Okay, this is getting out of hand. I've gone ahead and created a petutils library that includes the sort of directory tree's we're intent on testing on. It has it's own fixtures that use pytest's tmpdir. Going to just merge this and switch over directory creation to the fixtures contained in in openneuropet/petutils for further development.